From 2dc586f78e1d4c6991214d698418ae687d275271 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa@fedoraproject.org>
Date: Thu, 26 Sep 2024 09:48:51 -0400
Subject: [PATCH] gschema: Enable fractional scaling experimental features

Enabling these by default was the consensus in Workstation WG meetings[1][2].

[1]: https://pagure.io/fedora-workstation/issue/357#comment-848857
[2]: https://pagure.io/fedora-workstation/issue/357#comment-934865
---
 data/org.gnome.mutter.gschema.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.gnome.mutter.gschema.xml.in b/data/org.gnome.mutter.gschema.xml.in
index 6751a8d7a..d3a993c19 100644
--- a/data/org.gnome.mutter.gschema.xml.in
+++ b/data/org.gnome.mutter.gschema.xml.in
@@ -104,7 +104,7 @@
 
     <key name="experimental-features"
         flags='org.gnome.mutter.MetaExperimentalFeature'>
-      <default>[]</default>
+      <default>["scale-monitor-framebuffer", "xwayland-native-scaling"]</default>
       <summary>Enable experimental features</summary>
       <description>
         To enable experimental features, add the feature keyword to the list.
-- 
2.46.0

